domain/src/main/java/camp/nextstep/edu/domain/calculator/Calculator.kt

```diff
@@ -3,32 +3,28 @@ package camp.nextstep.edu.domain.calculator
 
 class Calculator {
 
-    fun evaluate(expression: String?): Int {
-        val parsed = ExpressionParser().parse(expression)
-        return calculateActual(parsed)
+    fun evaluate(expression: List<ExpressionItem>): Int {
+        ExpressionValidator().validOrError(expression)
+        return calculateActual(expression)
     }
 
-    private var pointer = 0
     private tailrec fun calculateActual(
         expressionItems: List<ExpressionItem>,
-        accumulator: List<ExpressionItem> = listOf()
+        acc: List<ExpressionItem> = listOf()
```

Comment:
```suggestion
        accumulator: List<ExpressionItem> = listOf()
```

왜 다시 축약어로 변경된 것일까요?ㅠㅠ